packages/scoutgame/src/notifications/appNotificationConstants.ts

```diff
@@ -0,0 +1,89 @@
+type Variables = {
```

Comment:
suggest renaming this to "templateVariables" since that's all that's really in here